html/index.html

```diff
@@ -17,11 +17,22 @@ <h1>Welcome to the Emoji Server</h1>
 			<tr>
 				<td colspan="2">
 					<form action="/search.html">
-					<input type="text" name="query" style="width: 80%; height: 36px"
-						placeholder="Type your desire here" />
-					<button class="button-text button green" type="submit">
-						Search
-					</button>
+						<div>
```

Comment:
why the extra divs?

Reply:
The extra div is an easy way to put the radio buttons below the search box,